### DISCUSSION TITLE
i don't know how to add an ID as an admin.

### DISCUSSION THREAD

**Discussion by labradf** [EvanMulawski/BeamMP.EnvSync#19] [Q&A] [answered]
I can't figure out how to add an ID as admin it just shows "You do not have permissions to do this" error. And do I need to sync the time in the chat? Because either way it doesn't seem to be actually syncing time even though it shows that it is syncing the time.
![image](https://user-images.githubusercontent.com/122172943/211163719-080d077e-6ca1-46ce-89ed-f948fa7806cc.png)
or is there maybe something that i am doing wrong?

**Comment by EvanMulawski** [comment, marked as answer]
Hi @labradf,

Regarding finding your ID to use for the admin list, please refer to https://github.com/EvanMulawski/BeamMP-EnvSync/discussions/17.

Regarding the time not syncing, you do not need to start the sync using chat - it should start automatically. I just confirmed this is working properly using BeamNG.drive 0.27.1, BeamMP Server 3.1.1, and BeamMP 4.7.1. If you bring up the console in BeamNG.drive using the tilde/backtick key, you should see this:

![image](https://user-images.githubusercontent.com/1044791/211166662-26fec1c1-98f4-429b-8ffa-b70fe7129966.png)

and ENVSYNC sync messages:

![image](https://user-images.githubusercontent.com/1044791/211166670-c520c2c6-a1c5-4edd-9889-b1b264ecce41.png)

**Reply by labradf** [reply]
Just checked it looks like it is just showing syncing time of day on the server cmd not beamng console. So I am probably doing something wrong. This time I didn't touch anything just put config in the server folder and dragged the 2 folders from src the client into client and server into server. I am using the server to just play with my friends so maybe you could send me the server with everything set up and working?

**Reply by EvanMulawski** [reply]
Make sure you download the [release](https://github.com/EvanMulawski/BeamMP-EnvSync/releases) and not the source code. The client mod must be a zip file.

**Reply by labradf** [reply]
ooooh thanks i think that i did that sorry imma try the release.

**Reply by labradf** [reply]
It works now thanks i still don't understand how to add an admin but it's ok

**Reply by EvanMulawski** [reply]
Your ID would be `328752`. The config entry would look like:

```
options.admins.beammp_328752 = "any description you want to put here"
```

**Reply by labradf** [reply]
oh, I think that I finally got everything to work thank you very much!
